src/element/iterators.rs

```diff
@@ -41,7 +41,7 @@ create_new_slice_iterator_type!(
     // Used for iterating over an Element's annotations
     SymbolsIterator => Symbol,
     // Used for iterating over a Sequence's Elements
-    ElementsIterator => Element
+    SequenceIterator => Element
```

Comment:
🗺️ There's another type (`ElementIterator`) that nearly clashes with `ElementsIterator`. I think `SequenceIterator` is a bit clearer anyway.